decidim-accountability/app/assets/stylesheets/decidim/accountability/accountability/_cards.scss

```diff
@@ -1,13 +1,15 @@
-.accountability {
```

Comment:
IMHO, spaces before the brackets look better. But maybe it's me!

Reply:
It looks better to me too, but it's the style of the current stylesheets, so I kept it to minimize the diff. Up to you though.